mapBuilder/controllers/default.classic.php

```diff
@@ -110,15 +116,21 @@ public function index()
         }
 
         // Write tree as JSON
-        $rep->addJSCode('var mapBuilder = {"layerStoreTree": '.json_encode($nestedTree).'};');
+        $this->fillJson('mapBuilder', 'layerStoreTree', $nestedTree);
 
         // Get original extent from ini file if set
         if (array_key_exists('extent', $readConfigPath)) {
-            $rep->addJSCode('mapBuilder.extent = ['.$readConfigPath['extent'].'];');
+            $extent = explode(',', $readConfigPath['extent']);
+
+            for ($i = 0; $i < count($extent); ++$i) {
+                $extent[$i] = (float) $extent[$i];
+            }
```

Comment:
```suggestion
            $extent = array_map('floatval', explode(',', $readConfigPath['extent']));
```
https://stackoverflow.com/questions/360725/how-can-i-convert-all-values-of-an-array-to-floats-in-php

Reply:
Changed in [7409ea6](https://github.com/3liz/lizmap-mapbuilder-module/pull/78/commits/7409ea6c3126c5abe887a0d6163d43d0c5674e44).